tests/js/spec/utils/dashboards/issueFieldRenderer.spec.tsx

```diff
@@ -133,4 +134,40 @@ describe('getIssueFieldRenderer', function () {
       expect(screen.getByText('Since issue began')).toBeInTheDocument();
     });
   });
+
+  it('can render annotations', async function () {
+    const renderer = getIssueFieldRenderer('annotations');
+
+    rtlMountWithTheme(
+      renderer!(data, {
+        location,
+        organization,
+      }) as React.ReactElement
+    );
+    expect(screen.getByText('ANNO-123')).toBeInTheDocument();
+  });
+
+  it('can render multiple annotations', async function () {
+    const renderer = getIssueFieldRenderer('annotations');
```

Comment:
in this case. what do you think we do something like:

```
 it('can render multiple annotations', function () {
    mountWithTheme(
      SPECIAL_FIELDS.annotations.renderFunc(
        {
          data,
          ...{
            annotations: [
              '<a href="sentry.io">ANNO-123</a>',
              '<a href="sentry.io">ANNO-456</a>',
            ],
          },
        },
        {
          location,
          organization,
        }
      ) as React.ReactElement
    );

    expect(screen.getByText('ANNO-123')).toBeInTheDocument();
    expect(screen.getByText('ANNO-456')).toBeInTheDocument();
  });
```

Reply:
I understand... the idea was not to need to use `renderer**!**` but I think it's fine